easy-note/cmd/api/.hz

```diff
@@ -0,0 +1,3 @@
+// Code generated by hz. DO NOT EDIT.
```

Comment:
use hz and idl in hertz api

Reply:
The IDL for the API is not visible, it should be a separate IDL.

Reply:
such as https://github.com/cloudwego/kitex-examples/tree/main/bizdemo/easy_note#api-requests